src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  box-sizing: border-box;
+  height: $card-height;
+  width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: $image-height;
+    width: $image-width;
```

Comment:
The variable `$image-width` is used for the `width` property of the `&__image` class. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid value before using it.